### DISCUSSION TITLE
After compiling and installing v0.10.0, tty cannot be switched

### DISCUSSION THREAD

**Discussion by Seryta** [neutrinolabs/xrdp#3101] [Q&A]
OS: ubuntu22.04
XRDP: v0.10.0
XORGXRDP: v0.10.1

I installed v0.10.0 of xrdp by manual compilation without uninstalling xrdp, and then I realized that ubuntu22 can't switch to other tty's via `ctrl+alt+f2~6` anymore (non-xrdp connection). And I can't view the tty info via `esc` when the system is booting, the page display stops at the graphical interface at that time.
I've tried a few things but none of them worked, such as completely uninstalling and emptying xrdp/xorgxrdp and then compiling and installing or apt installing, reinstalling xorg, etc. Can anyone help me?

**Comment by Seryta** [comment]
```shell
xorgxrdp git:v0.10.1 ❯ xrdp --version         
xrdp 0.10.0
  A Remote Desktop Protocol Server.
  Copyright (C) 2004-2024 Jay Sorg, Neutrino Labs, and all contributors.
  See https://github.com/neutrinolabs/xrdp for more information.

  Configure options:
      --enable-fuse
      --enable-mp3lame
      --enable-pixman

  Compiled with OpenSSL 3.0.2 15 Mar 2022
```

**Comment by matt335672** [comment]
This doesn't look like it can be an xrdp issue. We don't have anything to do with early system boot or the kernel.

Have you installed or updated any third party graphics drivers?

**Reply by Seryta** [reply]
No, I only tried to update after problems occurred, etc: `apt upgrade`.
And this machine is a virtual machine, there are no graphics card settings such as nvidia, only virtio_gpu

**Comment by Seryta** [comment]
I upgraded ubuntu22 to ubuntu24 and now everything looks fine!
